### ISSUE TITLE
Image Object Detection - Fine-Tuning

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#471]
- model selection
- learning rate
- anchors
- nms
- iou_threshold
- data augmention
- online hard example mining
- model 融合
- focal loss

**Comment by junxnone** [comment]
#435 #213
